example/src/CameraPage.tsx

```diff
@@ -250,13 +250,12 @@ export function CameraPage({ navigation }: Props): React.ReactElement {
                 isActive={isActive}
                 onInitialized={onInitialized}
                 onError={onError}
-                enableMetalPreview={enableMetalPreview}
                 enableZoomGesture={false}
                 animatedProps={cameraAnimatedProps}
                 photo={true}
                 video={true}
                 audio={hasMicrophonePermission}
-                syncFrameProcessor={device.supportsParallelVideoProcessing ? syncFrameProcessor : undefined}
+                syncFrameProcessor={device.supportsParallelVideoProcessing && enableFilter ? syncFrameProcessor : undefined}
```

Comment:
Do we still need the `device.supportsParallelVideoProcessing`?

Reply:
Nope - I'll remove that :)

Reply:
Well we don't have the Android changes in yet?